src/store/__tests__/RelayQueryTracker-test.js

```diff
@@ -57,7 +57,8 @@ describe('RelayQueryTracker', () => {
     tracker.trackNodeForID(query, 'client:1');
     const trackedChildren = tracker.getTrackedChildrenForID('client:1');
     expect(trackedChildren.length).toBe(1);
-    expect(trackedChildren[0])
+    expect(trackedChildren[0].getChildren().length).toBe(1);
+    expect(trackedChildren[0].getChildren()[0])
       .toEqualQueryNode(query.getFieldByStorageKey('actor'));
```

Comment:
This still doesn't pass. It seems the node I'm checking isn't of the right type (`Invariant Violation: printRelayOSSQuery(): Unsupported node type.`) but if I `console.log` the two objects are of the same type (and are actually the same).